liblustreapi/src/lib.rs

```diff
@@ -0,0 +1,237 @@
+// Copyright (c) 2019 DDN. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+use std::convert::From;
+use std::ffi::{CStr, CString};
+use std::fmt;
+use std::io;
```

Comment:
Won't -1 but I've been compressing common import statements into a single one.

I think there will be a formatter for this in the next version of rustfmt.